mregsite/settings.py

```diff
@@ -132,13 +138,14 @@
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
 
 REST_FRAMEWORK = {
     'DEFAULT_AUTHENTICATION_CLASSES': (
         'mreg.authentication.ExpiringTokenAuthentication',
     ),
     'DEFAULT_PERMISSION_CLASSES': (
-        'rest_framework.permissions.IsAuthenticated',
+        'mreg.api.permissions.IsInRequiredGroup',
```

Comment:
Could possible set it in local_settings.py instead. Though with REQUIRED_USER_GROUP unset it will allow any authenticated user...